app/src/main/java/at/roteskreuz/stopcorona/model/repositories/QuarantineRepository.kt

```diff
@@ -165,16 +195,25 @@ class QuarantineRepositoryImpl(
         by preferences.nullableZonedDateTimeSharedPreferencesProperty(PREF_DATE_OF_LAST_SELF_DIAGNOSE_BACKUP)
 
     private var dateOfLastRedContact: ZonedDateTime?
-        by preferences.nullableZonedDateTimeSharedPreferencesProperty(PREF_DATE_OF_LAST_RED_CONTACT)
+            by preferences.nullableZonedDateTimeSharedPreferencesProperty(
+                PREF_DATE_OF_LAST_RED_CONTACT
+            )
 
     private var dateOfLastYellowContact: ZonedDateTime?
-        by preferences.nullableZonedDateTimeSharedPreferencesProperty(PREF_DATE_OF_LAST_YELLOW_CONTACT)
+            by preferences.nullableZonedDateTimeSharedPreferencesProperty(
+                PREF_DATE_OF_LAST_YELLOW_CONTACT
```

Comment:
this break is also weird and might be connected with the codestyles

Reply:
Reverted the formatting